Langs/0C0A.lang

```diff
@@ -1,59 +1,64 @@
 [MetaData]
 Language=Spanish - Spain
-Translator=97carmine
+Translator=97carmine, JeffersonRussell
 Version=2.3.0.1
+[Font]
+Extra Large=24
+Large=18
+Medium=10
+Small=8.5
 [Strings]
 Donate=Donar
-Check for Updates=Comprobar actualizaciones
-Your Windows 11 Compatibility Results are Below=Los resultados de compatibilidad de Windows 11 están abajo 
-Now Reach WhyNotWin11 via https://www.whynotwin11.org/=Ahora llega a WhyNotWin11 a través de https://www.whynotwin11.org/
-Results Based on Currently Known Requirements!=¡Resultados basados en requisitos actualmente conocidos!
+Check for Updates=Actualizar
```

Comment:
My translation is more accurate

Reply:
Line 8: The sentence is too long.
Line 9: The beginning of the sentence does not make sense.
Line 10: I capitalized the same words as in the original sentence.

Reply:
Line 10: don't have to translate 1:1, follow your language's rules.

Reply:
If a translation 1:1 is not sought, then the translation is fine

Reply:
Thank you, I will merge this then. Feel free to submit later improvements